### DISCUSSION TITLE
Can you add a field like "Support URL" to an asset model

### DISCUSSION THREAD

**Discussion by lonelyadmin** [snipe/snipe-it#10193] [Q&A] [answered]
I'd like to have a support URL entry for the asset model. How can this be done?

**Comment by snipe** [comment, marked as answer]
Once v6 is out and custom fields can be applied to asset models, not just assets, this will likely be possible. In the meantime, you can use the support url field on Manufacturers.

**Reply by lonelyadmin** [reply]
Good to hear. Thanks!
